### ISSUE TITLE
angular-split is not accessible with keyboard navigation.

### ISSUE THREAD

**Issue by sandeeppatidar30** [angular-split/angular-split#98]
angular-split is not accessible using keyboard.

**Comment by beeman** [comment]
This has been fixed in https://github.com/angular-split/angular-split/pull/317

If there are still issues with keyboard accesibility, please open a new issue.

**Comment by Jefiozie** [comment]
@all-contributors please add @beeman for maintenance.

> 🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).
> Not all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.
> ...and of course, thank you for contributing! 💙
